<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTransactionDialog</class>
 <widget class="QDialog" name="AddTransactionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:  rgb(153, 193, 241)

</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Hash of recipient:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_hash">
     <property name="styleSheet">
      <string notr="true">background:  rgb(255, 255, 255)
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Value (in ZINAs, eg. 123.45)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_value">
     <property name="styleSheet">
      <string notr="true">background:  rgb(255, 255, 255)
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="input_message">
     <property name="styleSheet">
      <string notr="true">background:  rgb(255, 255, 255)
</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="Commit_button">
     <property name="styleSheet">
      <string notr="true">background:  rgb(26, 95, 180)
</string>
     </property>
     <property name="text">
      <string>Commit</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="CancelButton">
     <property name="styleSheet">
      <string notr="true">background:  rgb(26, 95, 180)
</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
